src/devices/Shtc3/README.md

```diff
@@ -0,0 +1,26 @@
+# SHTC3 - Temperature & Humidity Sensor
+SHTC3 is a digital humidity and temperature sensor designed especially for battery-driven high-volume consumer electronics application.
```

Comment:
empty line after title (otherwise in some special cases formatting might get broken when editing)